apps/src/lib/cli.rs

```diff
@@ -2449,8 +2447,8 @@ pub mod args {
             let broadcast_only = BROADCAST_ONLY.parse(matches);
             let ledger_address = LEDGER_ADDRESS_DEFAULT.parse(matches);
             let initialized_account_alias = ALIAS_OPT.parse(matches);
-            let fee_amount = GAS_AMOUNT.parse(matches);
-            let fee_token = GAS_TOKEN.parse(matches);
+            let fee_amount = FEE_AMOUNT.parse(matches);
```

Comment:
This will conflict with the bridge pool cli changes. There `Gas` is our gas and `Fee` is the relay fee.

Reply:
will update to be like as in https://github.com/anoma/namada/pull/737